pkg/kritis/cryptolib/pgp_test.go

```diff
@@ -86,7 +89,172 @@ N0Tk
 -----END PGP PUBLIC KEY BLOCK-----
 `
 
-const payload = `this is the payload i wish to sign
+const gpgPublicKeyID = "EE18BF76B63BE3B2B57DF36688BC30FC44D9AC5F"
+
+const gpgPrivateKey = `-----BEGIN PGP PRIVATE KEY BLOCK-----
+
```

Comment:
Why are there blank lines in the keys? This is not something I've seen before in PEM keys

Reply:
This is the output that gpg generated. When the blank lines are removed, `armor.Decode` fails with
```error decoding armored signature: EOF```